CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,35 @@
+# Contributer Code of Conduct
+
+## Sovereign Cloud Stack Community
+
+Sovereign Cloud Stack (SCS) is a network of existing and future providers of standardized sovereign cloud and container infrastructure to join forces in defining, implementing and operating a fully open, federated, compatible platform. We are committed to ["The Four Opens"](https://www.openstack.org/four-opens/) of the [Open Infrastructure Foundation](https://openinfra.dev/) and align our actions according to these four non-negotiable core principles of open collaboration. We do believe that an open and welcoming environment is essential for an active and engaged community. As contributors and maintainers we pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+No list is ever exhaustive, so we encourage members of the SCS community to adhere to the spirit, rather than the letter, of this code, as that is how it will be enforced. Places where this code may be particularly applicable are GitHub issues and pull requests, Matrix chat, mailing lists, discussions on social networks broadly directed at or between members of the community, and other direct interactions within the community. Any violations, especially continued or flagrant offenses, may affect an individual’s (or organization’s) ability to participate within the SCS community.
```

Comment:
> discussions on social networks broadly directed at or between members of the community

This, as written, emplaces the CoC even on private interactions of the community members, even if the interaction itself is not publicly visible at the time of the interaction. I note that this paragraph seems to be taken from the Ashai Linux CoC, and there they specifically point out Twitter instead of generally social networks, so it might be worth adding the word "public" here.

Reply:
If community members talk to each other on a non-public channel in abusive, harassing, insulting ways and one member brings it to the attention of the CoC Committee, it should feel empowered to act.

Reply:
Agreed, however, at least to me the wording is not limited to interactions between community members.

Reply:
We have generalized "Matrix chat" to chat. This also address violations that are sent private (from my understanding).